doc/en/reference.rst

```diff
@@ -1561,6 +1561,18 @@ passed multiple times. The expected format is ``name=value``. For example::
    See :ref:`change naming conventions` for more detailed examples.
 
 
+.. confval:: require_plugins
+
+   A space separated list of plugins that must be present for pytest to run.
+   If any one of the plugins is not found, emit a warning.
+   If pytest is run with ``--strict-config`` exceptions are raised in place of warnings.
+
+   .. code-block:: ini
+
+       [pytest]
+       require_plugins = pluginA pluginB pluginC
```

Comment:
Can we add here a real list? For example `require_plugins = pytest-xdist pytest-mock`?

I ask because it is not clear here if the user should add the package name or plugin name.